dogfood/Dockerfile

```diff
@@ -67,6 +67,8 @@ RUN mkdir --parents "$GOPATH" && \
     # go-releaser for building 'fat binaries' that work cross-platform
     go install github.com/goreleaser/goreleaser@v1.6.1 && \
     go install mvdan.cc/sh/v3/cmd/shfmt@latest
+		# nfpm is used with `make build` to make release packages
+		go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.16.0
```

Comment:
Not sure what happened there 😅

Reply:
Weird..looks fine in my editor? 

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/185453969-de847e89-a716-4e9b-b6e3-23e75a979217.png">

Reply:
`s/\t\/  /g`